cmd/thermal-recorder/snapshot.go

```diff
@@ -57,3 +60,46 @@ func newSnapshot(lastFrame int) (*cptvframe.Frame, error) {
 	}
 	return f, nil
 }
+
+func newSnapshotRecording() error {
+	mu.Lock()
+	defer mu.Unlock()
+
+	if processor == nil {
+		return errors.New("reading from camera has not started yet")
+	}
+
+	processor.StartSnapshot = true
+	return nil
+}
+
+func snapshotRecordingTriggers(processor *motion.MotionProcessor, window window.Window) {
+	if window.NoWindow {
```

Comment:
In this case, if the camera reconnects will the go routine keep running and also a new one?

Reply:
Have moved where the goroutine is made so this won't happen